src/render/decorations/CHyprGroupBarDecoration.cpp

```diff
@@ -179,6 +179,7 @@ CTitleTex::CTitleTex(CWindow* pWindow, const Vector2D& bufferSize) {
     const auto         CAIROSURFACE = cairo_image_surface_create(CAIRO_FORMAT_ARGB32, bufferSize.x, bufferSize.y);
     const auto         CAIRO        = cairo_create(CAIROSURFACE);
 
+    static auto const PTITLEFONTFAMILY = &g_pConfigManager->getConfigValuePtr("group:groupbar:font_family")->strValue;
```

Comment:
misses a `*` and a clang-format

Reply:
Apologies, I missed this comment. But I think this is done now.